pkg/rancher-desktop/assets/translations/en-us.yaml

```
@@ -2673,8 +2673,7 @@ prefs:
   experimental: Experimental
   onlyFromVentura_x64: This setting requires macOS 13.0 (Ventura) or later.
   onlyFromVentura_arm64: This setting requires macOS 13.3 (Ventura) or later.
-  onlyWithVZ_x64: This setting requires using the VZ emulation mode. VZ is only available on macOS 13.0 (Ventura) or later.
-  onlyWithVZ_arm64: This setting requires using the VZ emulation mode. VZ is only available on macOS 13.3 (Ventura) or later.
+  virtiofsDisabled: This setting has been temporarily removed to investigate a potential bug that could wipe out data on the host.
```

```suggestion
  virtiofsDisabled: This setting has been temporarily removed to investigate a potential bug that could lead to data loss on the host.
```
I'm okay with whichever phrasing.

---

No, this is better; I've updated it.